websites/A/AnimesGG/presence.ts

```diff
@@ -0,0 +1,139 @@
+const presence = new Presence({
+    clientId: "877352306100813944"
+  }),
+  pages: Record<string, PresenceData> = {
+    "/assinatura/": {
+      details: "Na pagina de premium",
+      state: "Vendo sobre os planos premiums"
+    },
+    "/login/": {
+      details: "Fazendo login no site",
+      state: "Login..."
+    },
+    "/perfil/": {
+      details: "Vendo seu proprio perfil",
+      state: "Vendo o perfil."
+    },
+    "/generos/": {
+      details: "Vendo todos os generos de anime.",
+      state: "Vendo..."
+    },
+    "/calendario/": {
+      details: "Vendo o calendario de animes",
+      state: "Vendo..."
+    }
+  };
+
+presence.on("UpdateData", async () => {
+  let data: PresenceData = {
+    largeImageKey: "logo",
+    startTimestamp: Math.floor(Date.now() / 1000)
+  };
+  const page = document.location.pathname,
+    params = new URLSearchParams(document.location.search.substring(1));
+
+  if (page in pages) 
+    data = Object.assign(data, pages[page]);
+  else if (page === "/") {
+    data.details = "Na pagina inicial";
+    data.state = "Procurando animes...";
+    if (params.get("s") !== null) {
+      data.details = "Procurando animes.";
+      data.state = `Procurando por ${params.get("s")}`;
+    }
+  } else if (page === "/lista-de-animes/") {
+    data.details = "Procurando animes.";
+    data.state = "Na lista de animes.";
+    if (params.get("search") !== "0" )
+      data.state = `Procurando por ${params.get("search")}`;
+  } else if (page === "/filmes/") {
+    data.details = "Procurando filmes de animes.";
+    data.state = "Na lista de filmes de animes.";
+    if (params.get("search") !== "0" && params.get("search") !== null)
+      data.state = `Procurando por ${params.get("search")}`;
+  } else if (page.includes("/anime/")) {
+    const isEpisode = page.includes("/episodio");
```

Comment:
You use it only once, I don't get why you have to make it a variable. To make it more readability? If that's the case then sure I guess